app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ActiveRecord::Base
 
-  has_many :questions, inverse_of: :asker
-  has_many :answers,   inverse_of: :answerer
```

Comment:
Yes it was came as is, i was thinking to remove extra spaces, but then left as is